src-tauri/src/domain/impls/tr17/section_2.rs

```diff
@@ -0,0 +1,36 @@
+pub mod equation_1;
+
+use serde::{Deserialize, Serialize};
+use specta::Type;
+
+#[derive(Clone, Type, Serialize, Deserialize, Debug)]
+pub enum Section2Equation {
+    One,
+}
```

Comment:
Not sure about this naming. Are we sure that we will only ever implement equations from Section 2? What if we string some equations together and make a method? Is that possible in this case?

```suggestion
#[derive(Clone, Type, Serialize, Deserialize, Debug)]
pub enum Section2Method {
    EquationOne,
}
```

Reply:
It is yes, and actually that is where the real value would be. I'll make the change.